middlewares/validators/items.js

```diff
@@ -0,0 +1,34 @@
+const Joi = require("joi");
+
+const { TYPES } = require("../../constants/items");
+
+const validateItemsPayload = async (req, res, next) => {
+  const schema = Joi.object({
+    itemid: Joi.string().trim().required(),
+    itemType: Joi.string()
+      .uppercase()
+      .custom((value, helper) => {
+        if (!TYPES.includes(value)) return helper.message("Not a valid type");
+        return value;
+      }),
+
+    tagPayload: Joi.array().items(
+      Joi.object({
+        tagid: Joi.string().required(),
+        levelid: Joi.string().required(),
+      })
+    ),
+  });
+
+  try {
```

Comment:
Add Joi block inside try block.

Reply:
No other files has this schema in try block, thus I followed them to maintain consistency